features/create_articles.feature

```diff
@@ -4,11 +4,15 @@ Feature: Create articles
   I would like to be able to create articles
 
   Background:
-    Given I visit the Homepage
+    Given the following user exists
+      | email           | password    | password_confirmation |
+      | harald@norge.no | OsloOslo123 | OsloOslo123           |
+    And I am logged in as "harald@norge.no"
```

Comment:
This seems to be present in another PR. Also, in what way is this relevant to the scope of this PR? (out-of-scope code makes code reviews harder, and I'm doing this in my free time)  Please check your commit history and make sure it's clean and not overlapping any other PR.